sig-federation/ONCALL.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+We have an oncall rotation in the SIG. The role description is as follows:
+
+* Ensure that the testgrid (https://k8s-testgrid.appspot.com/sig-federation) is green. This person will be the point of contact if testgrid turns red. Will identify the problem and fix it (most common scenarios: find culprit PR and revert it or free quota by deleting leaked resources).
```

Comment:
I think that we might want to extend this to monitoring the incoming queue of bug reports, and prioritizing and/or assigning the critical ones.  But we can debate that in the sig and do it in a followup PR as necessary.

Reply:
ack